config/fcos/v1_6_exp/validate.go

```diff
@@ -77,3 +77,14 @@ func (user GrubUser) Validate(c path.ContextPath) (r report.Report) {
 	}
 	return
 }
+
+func (m Module) Validate(c path.ContextPath) (r report.Report) {
+	if m.Name != "" && m.Content == "" {
+		r.AddOnError(c.Append("content"), common.ErrContentInvalid)
+	}
+
+	if m.Content != "" && m.Name == "" {
+		r.AddOnError(c.Append("name"), common.ErrNameInvalid)
+	}
```

Comment:
I don't follow the logic here. Why not just check both independently, possibly adding both errors if both fields are empty?

Reply:
I was trying not to generate any errors if both were empty in case the user doesn't fill in this. 

But your idea makes more sense. I'll change that. 

Thank you!